conda_smithy/templates/azure-pipelines-linux.yml.tmpl

```diff
@@ -17,6 +17,7 @@ jobs:
   - script: |
         export CI=azure
         export GIT_BRANCH=$BUILD_SOURCEBRANCHNAME
+        export FEEDSTOCK_NAME=`basename ${BUILD_REPOSITORY_NAME}`
```

Comment:
Please use `$(basename ...)` rather than backticks

Reply:
Happy to change! What is the difference? My bash-fu is weak.

Reply:
done